.circleci/config.yml

```
@@ -184,6 +185,15 @@ workflows:
             - "Build and bundle release artifacts (arm_macos)"
             - "Build and bundle release artifacts (amd_windows)"
           <<: *run_release
+  security-scans:
+    jobs:
+      - secops/gitleaks:
+          context:
+            - platform-docker-ro
+            - github-orb
+            - secops-oidc
+          git-base-revision: <<#pipeline.git.base_revision>><<pipeline.git.base_revision>><</pipeline.git.base_revision >>
```

why is there a `#` here?

```suggestion
          git-base-revision: << pipeline.git.base_revision >><< pipeline.git.base_revision>><</pipeline.git.base_revision >>
```

---

@EverlastingBugstopper Its been a minute since I've looked at this specific part, but if I'm remembering correctly, we needed the extra ugliness to ensure the job runs happily on merge/squash commits to the repo's default branch. 

I believe that when a CCI pipeline launches on main, the `<<pipeline.git.base_revision>>` parameter is not set. IIRC, Mustache (the templating language used by CCI) was happy to use the `#` and `/` syntax to test for the availability of the `pipeline.git.base_revision` parameter if it is null, but errored if you just included the parameter directly without those guards. 

I do have a card on my backlog to revisit all of this (because I too _really_ dislike looking at the templating nonsense), but it keeps getting bumped down the priority list.

---

makes sense to me! appreciate the explainer 😄